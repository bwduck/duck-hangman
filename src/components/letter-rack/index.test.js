import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import LetterRack from './index';

describe('<LetterRack/>', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(
      <LetterRack pickLetter={() => null} pickedLetters={['A', 'B']} />
    );
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
