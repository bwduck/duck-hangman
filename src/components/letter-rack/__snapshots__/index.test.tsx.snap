// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LetterRack/> renders and matches the snapshot 1`] = `
<div
  className="c-letter-rack"
>
  <LetterTile
    disabled={true}
    handleLetterTileClick={[Function]}
    key="A"
    letter="A"
  />
  <LetterTile
    disabled={true}
    handleLetterTileClick={[Function]}
    key="B"
    letter="B"
  />
  <LetterTile
    disabled={false}
    handleLetterTileClick={[Function]}
    key="C"
    letter="C"
  />
  <LetterTile
    disabled={false}
    handleLetterTileClick={[Function]}
    key="D"
    letter="D"
  />
  <LetterTile
    disabled={false}
    handleLetterTileClick={[Function]}
    key="E"
    letter="E"
  />
  <LetterTile
    disabled={false}
    handleLetterTileClick={[Function]}
    key="F"
    letter="F"
  />
  <LetterTile
    disabled={false}
    handleLetterTileClick={[Function]}
    key="G"
    letter="G"
  />
  <LetterTile
    disabled={false}
    handleLetterTileClick={[Function]}
    key="H"
    letter="H"
  />
  <LetterTile
    disabled={false}
    handleLetterTileClick={[Function]}
    key="I"
    letter="I"
  />
  <LetterTile
    disabled={false}
    handleLetterTileClick={[Function]}
    key="J"
    letter="J"
  />
  <LetterTile
    disabled={false}
    handleLetterTileClick={[Function]}
    key="K"
    letter="K"
  />
  <LetterTile
    disabled={false}
    handleLetterTileClick={[Function]}
    key="L"
    letter="L"
  />
  <LetterTile
    disabled={false}
    handleLetterTileClick={[Function]}
    key="M"
    letter="M"
  />
  <LetterTile
    disabled={false}
    handleLetterTileClick={[Function]}
    key="N"
    letter="N"
  />
  <LetterTile
    disabled={false}
    handleLetterTileClick={[Function]}
    key="O"
    letter="O"
  />
  <LetterTile
    disabled={false}
    handleLetterTileClick={[Function]}
    key="P"
    letter="P"
  />
  <LetterTile
    disabled={false}
    handleLetterTileClick={[Function]}
    key="Q"
    letter="Q"
  />
  <LetterTile
    disabled={false}
    handleLetterTileClick={[Function]}
    key="R"
    letter="R"
  />
  <LetterTile
    disabled={false}
    handleLetterTileClick={[Function]}
    key="S"
    letter="S"
  />
  <LetterTile
    disabled={false}
    handleLetterTileClick={[Function]}
    key="T"
    letter="T"
  />
  <LetterTile
    disabled={false}
    handleLetterTileClick={[Function]}
    key="U"
    letter="U"
  />
  <LetterTile
    disabled={false}
    handleLetterTileClick={[Function]}
    key="V"
    letter="V"
  />
  <LetterTile
    disabled={false}
    handleLetterTileClick={[Function]}
    key="W"
    letter="W"
  />
  <LetterTile
    disabled={false}
    handleLetterTileClick={[Function]}
    key="X"
    letter="X"
  />
  <LetterTile
    disabled={false}
    handleLetterTileClick={[Function]}
    key="Y"
    letter="Y"
  />
  <LetterTile
    disabled={false}
    handleLetterTileClick={[Function]}
    key="Z"
    letter="Z"
  />
</div>
`;
